vars/provisionNodes.groovy

```diff
@@ -254,10 +294,26 @@ EOF'''
                               ln -s python3.6 /usr/bin/python3
                           fi'''
     } else if (distro_type == "suse") {
-      if (repository != '') {
-        provision_script += '\nzypper --non-interactive ar ' +
-                            '--gpgcheck-allow-unsigned -f ' +
-                            repository + ' daos-stack-group-repo'
+      // Needed for sles-12.3/leap-42.3 only
+      provision_script += '\nrpm --import http://download.opensuse.org/' +
```

Comment:
Lack of SSL here is insecure as it's subject to a MitM attack.

Reply:
Fixed.  The http: URL is what is stored in the repository .repo file.